app/vmui/packages/vmui/src/utils/storage.ts

```diff
@@ -1,18 +1,18 @@
 export type StorageKeys = "AUTOCOMPLETE"
-    | "NO_CACHE"
-    | "QUERY_TRACING"
-    | "SERIES_LIMITS"
-    | "TABLE_COMPACT"
-    | "TIMEZONE"
-    | "DISABLED_DEFAULT_TIMEZONE"
-    | "THEME"
-    | "LOGS_LIMIT"
-    | "LOGS_MARKDOWN"
-    | "LOGS_DISABLED_HOVERS"
-    | "EXPLORE_METRICS_TIPS"
-    | "QUERY_HISTORY"
-    | "QUERY_FAVORITES"
```

Comment:
These changes update the key and structure of favorite queries in `localStorage`, so existing favorites will no longer be visible in the UI. I think it's worth adding a simple migration script if there's data under the old `QUERY_FAVORITES` key.

Reply:
Thanks, agree with you.
Added simple migration to the new structure.